ldscript.txt

```diff
@@ -104,39 +104,39 @@ SECTIONS
         src/malloc_vram.o(.text);
         src/input_recorder.o(.text);
 
-        /* sakit */
-        src/sakit/rings_manager.o(.text);
```

Comment:
IMO sa1 leftovers would still contain a "game" folder (since it's likely they just copied the whole sa1 codebase into the new sa2 project), otherwise the SA1 code wouldn't be so separate from other game data.

Reply:
I'd only worry about code that might match SA1 1:1 once we actually try merging it.

For now let's consider them separate entities.
Once we're at a stage where we can merge them all, we're gonna delete all redundancies we find, anyway.